api/src/pcapi/routes/backoffice_v3/templates/titelive/search_result.html

```diff
@@ -15,17 +15,17 @@ <h2 class="fw-light">Recherche EAN via Tite Live</h2>
                   {% if article.image %}
                     <img class="img-fluid rounded-start object-fit-contain"
                          src="{{ article.imagesUrl.recto }}"
-                         alt="Illustration pour {{ json.oeuvre.titre }}"
-                         title="Illustration pour {{ json.oeuvre.titre }}" />
+                         alt="Illustration pour {{ json.oeuvre.titre|safe }}"
+                         title="Illustration pour {{ json.oeuvre.titre|safe }}" />
```

Comment:
(question) Pourquoi veut-on utiliser `safe` ici (et pour d'autres champs) ? Ce champ ne devrait pas contenir de HTML et, si jamais il en contient, l'échappement nous évitera un trou de sécurité, non ?

Reply:
Je suis tombé sur des résumé et titre avec par exemple `&#34`, ceci permet de les rendre lisible.

Reply:
Je précise que sur le titre, je n'ai pas eu de cas, mais dans le doute, je préfère le mettre.

Reply:
Je ne suis pas super convaincu : on ne contrôle pas ce que l'on reçoit (et apparemment Tite Live ne maîtrise pas bien les choses non plus, ou alors reçoit lui-même des données moisies). Il faut peser le pour et le contre : 
- en utilisant `| safe`, on a un risque d'injection JavaScript. Le risque est sûrement faible, mais il n'est pas nul.
- en n'utilisant **pas** `| safe`, on élimine ce risque. L'inconvénient est qu'on aura parfois des `&quote;` et autres entités HTML affichés en toutes lettres, mais je pense qu'on peut vivre avec. *Garbage in, garbage out.* :)

Quelqu'un d'autre aurait un avis sur la question ? @prouzet , @Meewan , @Vencespass , @francois-seguin ?

(Si tu décides de faire cette modification, ce serait bien qu'elle soit dans un commit séparé, avec un message de commit indiquant que tu es tombé sur ce cas de `&quote;`.)

Reply:
On peut juste utiliser safe puis retirer certains char ou les remettre en html entities et faire un parser dédié ?

Reply:
Mon avis va plutôt dans le sens de Damien : ne pas utiliser `| safe` sur une variable dont on ne maîtrise pas totalement le contenu. Si c'est uniquement pour les _html entities_, alors créons un filtre `unescape` appelé à la place de `safe`, et qui est connecté sur la fonction Python `html.unescape`

Reply:
Ok pour moi !